modules/stanford_profile_styles/lib/scss/paragraph-types/wysiwyg/_buttons.scss

```diff
@@ -17,4 +17,14 @@
     @include modular-spacing('margin-bottom', 0);
   }
 
+  // If they're in the same paragraph, get them to stack.
+  [class*="su-button"] {
+    display: block;
+    width: max-content;
```

Comment:
Wouldn't this force all buttons of any type to take up the entire line? As in authors would not be able to but two buttons side by side? I don't think we want to force that do we?

Reply:
Better check with Kerri/Ishi.

Reply:
I ran a quick fiddle: https://jsfiddle.net/j95gqb8a/

Reply:
Hi @sherakama, 
Thanks for the comment.
Yes, I have brought it up with @kerri-augenstein, but not Ishi.

We could add a style to the WYSIWYG to allow stacking or visa versa.

The question you raise is, what is the default? 
When two buttons are next to each other in the WYSIWYG should they stack or should they site next to each other in the same line?

Reply:
Hi @sherakama,

I'm wondering if this will work? I created a css class `su-inline-button` that the editor  can add to the `<p>` through the WYSIWYG.

In the WYSIWYG, this is what it will look like:
```<p class="su-inline-button">
  <a class="su-button--big" href="https://amptesting.sites.stanford.edu/">button 1</a> 
  <a class="su-button--secondary" href="https://amptesting.sites.stanford.edu/">button 2</a> 
  <a class="su-button" href="https://amptesting.sites.stanford.edu/">button 3</a>
  <a class="su-button--big" href="https://amptesting.sites.stanford.edu/">button 1</a> 
  <a class="su-button--secondary" href="https://amptesting.sites.stanford.edu/">button 2</a> 
  <a class="su-button" href="https://amptesting.sites.stanford.edu/">button 3</a>
</p>```

Reply:
Thanks for this update. This PR is on my radar to review but I have a few other items stacked up before this.